### ISSUE TITLE
Is there a mechanism to translate the tags attached to a question? 

### ISSUE THREAD

**Issue by litebluecheese** [lucianferoiu/youvote4eu#19]
I suppose not. A solution could be to attach the tags during translation, not at publish time. Like this they will be translated together with the question text.
As a separate note, I'm still not a fan of the tags solution... :-( Everyone can write there any shit. I imagine when the list will be long, some people will not look to see if there is already something similar, they will keep adding new words.

**Comment by lucianferoiu** [comment]
No, currently there is no support in the UI for adding or translating tags...  (for now, in dev it's possible to add a new tag in you write it, but it is meant to be temporary - we should definitely avoid the tags inflation)
